models/executor.js

```diff
@@ -5,4 +5,28 @@ module.exports = function(Executor) {
     }
     next();
   };
+
+  Executor.disableRemoteMethod('__delete__instances');
```

Comment:
I mean, why are any of these being disabled? We have REST API, they use it, what problem are we solving?